src/swagger-generation/output/microsoftgraph-beta-0.1.8-preview.json

```
@@ -1456,7 +1466,7 @@
         ],
         "responses": {
           "200": {
-            "description": "group created/updated successfully",
+            "description": "group create or update successfully",
```

```suggestion
            "description": "Group created or updated successfully",
```
Same in other places